common/plugins/eu.esdihumboldt.hale.common.headless/src/eu/esdihumboldt/hale/common/headless/transform/ValidationJob.java

```diff
@@ -71,53 +73,63 @@ public ValidationJob(InstanceValidator validator, ReportHandler reportHandler,
 	 */
 	@Override
 	protected IStatus run(IProgressMonitor monitor) {
-		IOReporter defaultReporter = validator.createReporter();
-		defaultReporter.setSuccess(false);
-		IOReport report = defaultReporter;
-		try {
-			ATransaction trans = log.begin(defaultReporter.getTaskName());
+		boolean successful = true;
+
+		for (InstanceValidator validator : this.validators) {
+			IOReporter defaultReporter = validator.createReporter();
+			defaultReporter.setSuccess(false);
+			IOReport report = defaultReporter;
 			try {
-				if (writer != null) {
-					// set validation schemas (may have been determined only
-					// during writer execution)
-					List<? extends Locatable> schemas = writer.getValidationSchemas();
-					validator.setSchemas(schemas.toArray(new Locatable[schemas.size()]));
-				}
-				IOReport result = validator.execute(new ProgressMonitorIndicator(monitor));
-				if (result != null) {
-					report = result;
-				}
-				else {
-					defaultReporter.setSuccess(true);
+				ATransaction trans = log.begin(defaultReporter.getTaskName());
+				try {
+					if (writer != null) {
+						// set validation schemas (may have been determined only
+						// during writer execution)
+						// set schemas
+						List<? extends Locatable> schemas = writer.getValidationSchemas();
+						validator.setSchemas(schemas.toArray(new Locatable[schemas.size()]));
+					}
+					IOReport result = validator.execute(new ProgressMonitorIndicator(monitor));
+					if (result != null) {
+						report = result;
+					}
+					else {
+						defaultReporter.setSuccess(true);
+					}
+				} catch (Throwable e) {
+					defaultReporter.error(new IOMessageImpl(e.getLocalizedMessage(), e));
+				} finally {
+					trans.end();
 				}
 			} catch (Throwable e) {
 				defaultReporter.error(new IOMessageImpl(e.getLocalizedMessage(), e));
-			} finally {
-				trans.end();
 			}
-		} catch (Throwable e) {
-			defaultReporter.error(new IOMessageImpl(e.getLocalizedMessage(), e));
-		}
 
-		if (monitor.isCanceled()) {
-			reset();
-			return Status.CANCEL_STATUS;
-		}
+			if (monitor.isCanceled()) {
+				reset();
+				return Status.CANCEL_STATUS;
+			}
+
+			// add report to report service
+			reportHandler.publishReport(report);
 
-		// add report to report service
-		reportHandler.publishReport(report);
+			// show message to user
+			if (report.isSuccess()) {
+				// info message
+				log.userInfo(report.getSummary());
```

Comment:
If run in the UI, when mutiple validators are used, this will lead to multiple dialogs being displayed to the user, each after one validator succeeded. It might be better to log normal messages here (e.g. `info` instead of `userInfo`) and provide a general summary requesting the user's attention after all validators completed.